Hardware info/script3.sh

```diff
@@ -0,0 +1,18 @@
+
+#!bin/bash
```

Comment:
the interpreter is located here: `/bin/bash'
so update this line to `#!/bin/bash`